src/demo/signup-screen/signup.screen.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+  View,
+  StyleSheet,
+  Text,
+  TextInput,
+  ImageBackground,
+} from 'react-native';
+
+import AuthenticationSection from './components/authentication-section';
+import Header from './components/header';
+import Footer from './components/footer';
+
+const LivingRoomScreen: React.FC = () => {
+  return (
+    <ImageBackground
+      source={require('./assets/signup_background.png')}
+      style={styles.root}>
+      <Header name="chevron-left" color="white" />
+      <View style={styles.heading}>
+        <Text style={styles.title}>Create</Text>
+        <Text style={styles.title}>Account</Text>
+      </View>
+      <View style={styles.inputSection}>
+        <TextInput
+          style={styles.textInput}
+          placeholderTextColor="white"
+          placeholder="Name"
+        />
+        <TextInput
+          style={styles.textInput}
+          placeholderTextColor="white"
+          placeholder="Email"
+          textContentType="emailAddress"
+        />
+        <TextInput
```

Comment:
in react native please wrap text input inside into keyboard avoiding view for better user experience